From 713212d36762a325b1d9c3fb0b0642bd2e333d9d Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Mon, 24 Feb 2025 20:31:41 +0100
Subject: [PATCH 1303/1695] dts: stm32: remove stm32mppll3-cfg-900Mhz on
 STM32MP23 board

On STM32MP23 Soc the GPU is limited at 400MHz so remove the 900MHz
not supported by SoC.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git e3829598f156ea1d3bbe5be7c11b1fd2e87ef3a1 dts: stm32: remove stm32mppll3-cfg-900Mhz on STM32MP23 board]
Fixes: ddad7d553b0f ("dts: stm32: add STM32MP23 RCC and RIF configuration for DK board")
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: I8873f5375b4b6fcac6b94b790d0001cd490d7d7b
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/439607
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/arch/arm/dts/stm32mp235f-dk-ca35tdcid-rcc.dtsi | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp235f-dk-ca35tdcid-rcc.dtsi b/core/arch/arm/dts/stm32mp235f-dk-ca35tdcid-rcc.dtsi
index 50a7c6e79..eeeeb314e 100644
--- a/core/arch/arm/dts/stm32mp235f-dk-ca35tdcid-rcc.dtsi
+++ b/core/arch/arm/dts/stm32mp235f-dk-ca35tdcid-rcc.dtsi
@@ -151,11 +151,6 @@
 			cfg = <20 1 1 2>;
 			src = <MUX_CFG(MUX_MUXSEL7, MUXSEL_HSE)>;
 		};
-
-		pll3_cfg_900Mhz: pll3-cfg-900Mhz {
-			cfg = <45 2 1 1>;
-			src = <MUX_CFG(MUX_MUXSEL7, MUXSEL_HSE)>;
-		};
 	};
 
 	pll4: st,pll-4 {
-- 
2.47.2

